kolibri/utils/cli.py

```diff
@@ -14,10 +15,44 @@
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
+from kolibri import dist_cext as kolibri_dist_cext
+
 sys.path = sys.path + [
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
 ]
 
+
+"""
+Get the directory of dist_cext.
+"""
```

Comment:
Seeing that you have written a docstring for a function, I would suggest moving all this to a function to keep the module more readable.